pravega/pravega
# Allow the REST API's serverPort be set by an enviroment variable if

Possible fix for #482

**Purpose of the change**

Allows the REST serverPort to be configured to something besides port 80, defaults to port 80.

I found the way to set a default/optional environment variables from https://github.com/typesafehub/config#optional-system-or-env-variable-overrides.

**How to verify it**

Run it without setting the environment variable. You should see it on port 80. Run it with REST_PORT set and you'll see it on whatever you set it as.
